Cargo.toml

```
@@ -47,19 +47,19 @@ version = "0.6.0"
 path = "sdk/typespec/typespec_macros"
 
 [workspace.dependencies.azure_core]
-version = "0.27.0"
+version = "0.28.0"
 path = "sdk/core/azure_core"
 
 [workspace.dependencies.azure_core_macros]
-version = "0.1.0"
+version = "0.2.0"
 path = "sdk/core/azure_core_macros"
 
 [workspace.dependencies.azure_core_opentelemetry]
 version = "0.1.0"
```

This didn't get updated for some reason, though the other files did.

```suggestion
version = "0.2.0"
```